### PR TITLE
Deps: Batch @types renovate updates

### PR DESCRIPTION
**Pull request by sirreal**
Try batching @types dependency updates. They are generally safe and can be merged without issue. May help keep the volume of renovate down.

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -31,6 +31,10 @@
 			"packageName": "typescript",
 			"prPriority": 1
 		},
+		{
+			"packagePatterns": [ "^@types/" ],
+			"prPriority": 1
```
Comment (jsnajdr) [inline renovate.json]: Maybe there should be also `groupName` field that tells Renovate to group the packages into one update and give the group a name?

File: renovate.json [hunk 2]
```diff
@@ -31,6 +31,10 @@
 			"packageName": "typescript",
 			"prPriority": 1
 		},
+		{
+			"packagePatterns": [ "^@types/" ],
```
Comment (sirreal) [inline renovate.json]:
```suggestion
			"groupName": "Type definitions",
			"packagePatterns": [ "^@types/" ],
```

File: renovate.json (hunk 1 above)
Comment (sirreal) [reply renovate.json]: Good plan, added `Type definitions`.

**Review by jsnajdr — APPROVED** [review]
Thanks, looks like something that could work 🙂 👍

**Pull request merged**